docs/publish/deploy-office-add-in-sso-to-azure.md

```diff
@@ -0,0 +1,253 @@
+
+## Requirements
+
+The steps in this article work for an Office Add-in created by the [Yeoman Generator for Office Add-ins](https://github.com/OfficeDev/generator-office) using the `Office Add-in Task Pane project supporting single sign-on (localhost)` project type. Be sure you have configured the add-in project so that it runs on localhost successfully. For more information, see the [Single sign-on (SSO) quick start](../quickstarts/sso-quickstart.md).
+
+The steps in this article also require:
+- [Azure Account extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.azure-account) for VS Code.
+- [Azure App Service extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice) for VS Code.
+
+## Create the app service
+
+The following steps set up a basic deployment of the Office Add-in. There are multiple ways to configure deployment that are not covered in this documentation. For additional options on how you may want to configure your deployment, see [Deployment Best Practices](/azure/app-service/deploy-best-practices)
+
+1. Open your Office Add-in project in VS Code.
+1. Select the Azure logo in the [Activity Bar](https://code.visualstudio.com/docs/getstarted/userinterface). If the Activity Bar is hidden, open it by selecting **View** > **Appearance** > **Activity Bar**.
+1. In the **App Service** explorer, select **Sign in to Azure...** and follow the instructions.
+
+    :::image type="content" source="../images/azure-extension-sign-in.png" alt-text="Sign in to Azure button selected in the Azure extension.":::
+
+# [Deploy to Linux](#tab/linux)
+
+1. Right-click on App Services and select **Create new Web App**. A Linux container is used by default.
+1. Type a globally unique name for your web app and press **Enter**. The name must be unique across all of Azure and use only alphanumeric characters ('A-Z', 'a-z', and '0-9') and hyphens ('-').
+1. In Select a runtime stack, select the **Node 16 LTS** runtime stack.
+1. In Select a pricing tier, select **Free (F1)** and wait for the resources to be provisioned in Azure. When prompted to deploy, don't deploy the add-in yet. You'll do that in a later step.
+
+# [Deploy to Windows](#tab/windows)
+
+1. Right-click on App Services and select **Create new Web App... Advanced**.
+1. Type a globally unique name for your web app and press **Enter**. The name must be unique across all of Azure and use only alphanumeric characters ('A-Z', 'a-z', and '0-9') and hyphens ('-').
+1. Select the resource group you wan to use. If you don't have a resource group, select **Create a new resource group**, then enter a name for the resource group, such as *AppServiceQS-rg*.
+1. Select the **Node 16 LTS** runtime stack.
+1. Select **Windows** for the operating system.
+1. Select the location you want to serve your app from. For example, *West Europe*.
```

Comment:
I got this error the first time with West US 2. A different region worked. Maybe it's because the account is new?
![image](https://user-images.githubusercontent.com/38896772/209880898-66480001-46d7-47da-b6ea-714a49a6d7ff.png)

Reply:
It looks like this could be region specific, or could depend on your subscription and what you've already allocated (free is probably more limited)? https://learn.microsoft.com/en-us/azure/quotas/view-quotas